web_environment_ribbon/static/src/css/ribbon.css

```diff
@@ -16,7 +17,7 @@
     z-index: 1000;
     position: fixed;
     box-shadow: 0 0 3px rgba(0,0,0,.3);
-    background: rgba(255, 0, 0, .6);;
+    background: rgba(255,0,0,.6);
```

Comment:
Following guidelines to migrate v8 to v9 (), I've respected commits merged in v8 branch. Please @jabibi answer this question.